src/MyProblematicClass.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+class MyProblematicClass
+{
+    protected string $test;
+
+    public function test(): string
+    {
+        foreach ($_POST['menu-locations'] as $menu_id) {
```

Comment:
:warning: **Warning**: Detected usage of a non-sanitized input variable: $_POST['menu-locations'] (*WordPress.Security.ValidatedSanitizedInput.InputNotSanitized*).